src/main/java/io/cdap/plugin/salesforce/connector/SalesforceConnector.java

```diff
@@ -129,7 +129,9 @@ public ConnectorSpec generateSpec(ConnectorContext connectorContext, ConnectorSp
     }
     AuthenticatorCredentials authenticatorCredentials = config.getAuthenticatorCredentials();
     try {
-      SObjectDescriptor sObjectDescriptor = SObjectDescriptor.fromName(tableName, authenticatorCredentials);
+      String fields = getObjectFields(tableName);
```

Comment:
What is this change for ?

Reply:
This change is required for the schema purpose, to limit the number of fields in schema.